EncodingStreamingHighAvailability/HighAvailability.Provisioning/ProvisioningFunction.cs

```diff
@@ -0,0 +1,57 @@
+namespace HighAvailabikity.Provisioner
+{
+    using HighAvailability.Interfaces;
+    using HighAvailability.Models;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using System;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Implements provisioning Azure function. It is triggered by messages in provisioning-requests Azure queue
```

Comment:
Needs more details. I should easily be able to understand the need for this function and how it fits in the Sample

Reply:
Will add link to final document